cmd/bnsd/app/testdata/fixtures/app.go

```diff
@@ -137,7 +137,7 @@ func appStateGenesis(keyAddress weave.Address) []byte {
 		"escrow": []interface{}{
 			dict{
 				"sender":    "0000000000000000000000000000000000000000",
-				"arbiter":   "multisig/usage/0000000000000001",
+				"arbiter":   weave.Condition("cond:multisig/usage/0000000000000001").Address(),
```

Comment:
[`cond` prefix is used by JSON deserializer](https://github.com/iov-one/weave/blob/master/conditions.go#L170-L179). When building a condition manually you should not prefix it. You can use [`NewCondition`](https://github.com/iov-one/weave/blob/master/conditions.go#L33) to build one.

Reply:
Yeah, I know. It was a "clever" hack to make it pass, I'll fix that up properly now.